bootstrap/bootstrap/bootstrap.py

```diff
@@ -59,6 +60,55 @@ def read_config_file() -> Dict[str, Any]:
         }
         return config
 
+    def pull_core(self) -> None:
+        curses_ui = True
+        try:
+            # TODO: make it not clear the whole screen # pylint: disable=fixme
+            screen = curses.initscr()
+            curses.noecho()
+            curses.cbreak()
```

Comment:
I would recommend to set `curs_set(0)` to turn the cursor invisible and remove the weird effect of the cursor moving around the screen.